deps/libuv.version

```diff
@@ -1,2 +1,2 @@
-LIBUV_BRANCH=julia-uv1.9.0
-LIBUV_SHA1=ecbd6eddfac4940ab8db57c73166a7378563ebd3
+LIBUV_BRANCH=jn/pid-zero
+LIBUV_SHA1=cb6d0f875a5b8ca30cba45c0c1ef7442c87c1e68
```

Comment:
the work is done and ready for merging. this just requires slightly more manual work than the usual Github button. however, we don't consider the merging "work" in the WIP tag.

(aside: I find it strange when there is both a reply and reaction on a comment. aren't they conveying redundant information?)